npm/react/cypress/component/advanced/mocking-axios/2-users-api.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState, useEffect } from 'react'
+// import wrapped Axios method
+import axiosApi from './axios-api.jsx'
+
+export function Users () {
+  const [users, setUsers] = useState([])
+
+  const getUsers = async () => {
+    console.log({ axiosApi })
+    const response = await axiosApi.get('https://jsonplaceholder.cypress.io/users?_limit=3')
+    // JSON responses are automatically parsed.
```

Comment:
This comment seems misplaced/not sure it's really needed.

Reply:
addressed in 90da84a3b5